packages/server/lib/environment.coffee

```diff
@@ -39,6 +40,15 @@ try
   if os.platform() is "linux"
     app.disableHardwareAcceleration()
 
+  if process.env.ELECTRON_EXTRA_LAUNCH_ARGS
+    electronLaunchArgs = parseElectronLaunchArguments(process.env.ELECTRON_EXTRA_LAUNCH_ARGS)
+    Object.keys(electronLaunchArgs).forEach (key) ->
+      value = electronLaunchArgs[key]
+      if value == undefined
+        app.commandLine.appendSwitch(key)
+      else
+        app.commandLine.appendSwitch(key, value)
```

Comment:
I think most of this complexity could go away if you used `app.commandLine.appendArgument`, you wouldn't need to split `key` and `value`, you could just pass it directly to `appendArgument` as one complete arg.

Reply:
https://electronjs.org/docs/api/command-line#commandlineappendargumentvalue

Reply:
Yeah, but they suggest using `appendSwitch` :)
```
f you're appending an argument like --switch=value, consider using appendSwitch('switch', 'value') instead.
```

Reply:
that's because they're assuming you have `switch` and `value` already, which in our case, we don't

Reply:
well, one could pass both combinations I think

Reply:
let's remove the extra complexity from our codebase and just use `.appendArgument` with the original args the user passed

Reply:
done - switched to `appendArgument`